modules/alloy/src/main/ts/ephox/alloy/behaviour/transitioning/TransitionApis.ts

```diff
@@ -46,17 +46,21 @@ const disableTransition = (comp: AlloyComponent, transConfig: TransitioningConfi
 
 const getNewRoute = (comp: AlloyComponent, transConfig: TransitioningConfig, transState: Stateless, destination: string): TransitionRoute => {
   return {
-    start: Attr.get(comp.element(), transConfig.stateAttr),
+    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+    start: Attr.get(comp.element(), transConfig.stateAttr)!,
```

Comment:
Wasn't quite too sure what to do here, as it's possible for the attribute to not exist, meaning it'll return `null`. Given we never checked anything before and there's not much better we can do here, I figured asserting it won't be null should be okay.